File: NOTES/SelectionSort.java
public class SelectionSort {
    public static void main (String[] args) {
        int[] nums = {4, 1, 6, 5, 2, 9, 8, 7, 8, 9, 3, -1};

        SelectionSort(nums);
        for (int num : nums) {
            System.out.print(num + " ");
        }


    }

    static void SelectionSort(int[] list) {
        for (int i = 0; i < list.length; i++) {
            for (int j = i+1; j < list.length; j++) {
                if (list[j] < list[i]) {
                    int temp = list[j];
                    list[j] = list[i];
                    list[i] = temp;
                }
            }
        }
    }
}

File: NOTES/selectionsort.py
def selectionsort(list):
    for i in range(len(list)):
        for j in range(i+1, len(list)):
            if list[j] < list[i]:
                list[j], list[i] = list[i], list[j]
    
    return list

nums = [4, 6, 3, 9, 2, 1]
print(selectionsort(nums))